frontend/common/hydrogen.config.json

```diff
@@ -2,7 +2,8 @@
   "debug": false,
   "mediaWarnings": false,
   "folders": {
-    "scripts": "src",
+    "markup": "resources/views",
```

Comment:
Are we missing anything by losing `src`? I didn't understand this part, but I think @JoshBeveridge said it's not possible to use two directories here (but will be possible in the next major release)?

## Before

<details>
<summary>Click to see code fragments...</summary>

```json5
// frontend/admin/hydrogen.config.json
{
  // ...
  "markup": "resources/views",
  // ...
}
```

```json5
// frontend/common/hydrogen.config.json
{
  // nothing
}
```

```json5
// frontend/talentsearch/hydrogen.config.json
{
  // ...
  "markup": "resources/views",
  // ...
}
```

</details>


### Now

<details>
<summary>Click to see code fragments...</summary>

```json5
// frontend/common/hydrogen.config.json
{
  // ...
  "markup": "resources/views",
  // ...
}
```

</details>